.github/workflows/nightly.yml

```diff
@@ -49,14 +49,12 @@ jobs:
           SHORT_SHA=${SHA::7}
 
           # get latest nightly tag
-          LATEST_NIGHTLY_TAG=$(git tag -l v0.0.0-nightly-\* --sort=-taggerdate | head -n 1)
+          LATEST_NIGHTLY_TAG=$(git tag -l v0.0.0-nightly-\* --sort=-committerdate | head -n 1)
 
-          # check if last commit to dev starts would be the nightly tag we're about to create (minus the date)
-          # if it is, we'll skip the nightly creation
-          # if not, we'll create a new nightly tag
-          if [[ ${LATEST_NIGHTLY_TAG} == v0.0.0-nightly-${SHORT_SHA}-* ]]; then
-            echo "🛑 Latest nightly tag is the same as the latest commit sha, skipping nightly release"
-          else
+          CHANGES=$(git diff $LATEST_NIGHTLY_TAG..dev -- ./packages/ -- ':!packages/**/package.json')
```

Comment:
check for changes to `packages/*` ignoring updates to `package.json` due to the version making the file change